docs/decimal128.md

```diff
@@ -0,0 +1,59 @@
+## Decimal128
+
+You can define a Decimal128 type property as
+
+```js
+// in model.json file
+"aproperty": {
+  "type": "String",
+  "mongodb": {"dataType": "Decimal128"}
```

Comment:
Out of scope of this pull request: it would be great to support `decimal` and/or `numeric` as native LoopBack types. AFAICT, most SQL databases support Decimal and Numeric types too, see e.g.

- [PostgreSQL](https://www.postgresql.org/docs/9.1/static/datatype-numeric.html )
- [Microsoft SQL](https://docs.microsoft.com/en-us/sql/t-sql/data-types/precision-scale-and-length-transact-sql?view=sql-server-2017)
- [Oracle](https://docs.oracle.com/javadb/10.6.2.1/ref/rrefsqlj15260.html)
- [MySQL](https://dev.mysql.com/doc/refman/5.7/en/precision-math-decimal-characteristics.html)
- [db2](https://www.ibm.com/support/knowledgecenter/en/SSEPEK_10.0.0/sqlref/src/tpc/db2z_bif_decimal.html)

So ideally, a decimal property should be defined as follows:

```js
// in model.json file
"aproperty": {
  "type": "decimal",
}
```

Thoughts?

cc @raymondfeng

Reply:
@bajtos Good point. I would suggest add it in the new juggler or `@loopback/types`, I remember we have a PR to refactor built-in types to a new package.

Reply:
Makes sense. I created a new issue to keep track of my feature request, see  https://github.com/strongloop/loopback-next/issues/1902